src/Faker/Provider/Text.php

```diff
@@ -40,7 +61,31 @@ public function realText($maxNbChars = 200, $indexSize = 2)
             throw new \InvalidArgumentException('indexSize must be at most 5');
         }
 
+        if ($minNbChars >= $maxNbChars) {
```

Comment:
Shouldnt we set a minimum here as well? And check for valid values since an -10 can also be passed

Reply:
I've now added a check that `$minNbChars` is not smaller than 1. 

It doesn't make much sense to set `$minNbChars` to something much smaller than `$maxNbChars`, but it also doesn't prevent the algorithm from working as expected (producing real text that is between min and max chars long), so I wouldn't prohibit doing it.